packages/docusaurus/src/server/plugins/__tests__/__fixtures__/simple-site/docusaurus.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  title: 'My Site',
+  tagline: 'The tagline of my site',
+  url: 'https://your-docusaurus-test-site.com',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  plugins: [function (context, params) { }, [function (context, params) { }, { it: 'should work' }]]
```

Comment:
while we are working on it, maybe also add a plugin using a string path, so that we cover all the cases in the test